File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

func Convert(number int) string {
    out := ""
	if number % 3 == 0{
        out = "Pling"
    }
    if number % 5 == 0 {
        out += "Plang"
    }
	if number % 7 == 0 {
        out += "Plong"
    }
    if out == "" {
        out = strconv.Itoa(number)
    }
    return out
}

File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number)
    sound = ""
    if number % 3 == 0
        sound *= "Pling"
    end
    if number % 5 == 0
        sound *= "Plang"
    end
    if number % 7 == 0
        sound *= "Plong"
    end
    if sound == ""
        sound = string(number)
    end
    sound
end
